index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+   <title>Alan Turing - Past and Future</title>
+   <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+
+<body>
+  <nav>
+    <strong>Past Achievements</strong>
+     * 
+    <a href="future_goals.html">Future Goals</a>
+  </nav>
+
+  <h1>Alan <strong>M.F.</strong> Turing</h1>
```

Comment:
Use of presentational tags such as `strong` should be avoided when possible in favor of a CSS class, which is current best practice.  🔨  

Please read this article on specificity (https://css-tricks.com/specifics-on-css-specificity/) and BEM (https://csswizardry.com/2011/09/when-using-ids-can-be-a-pain-in-the-class/).  BEM is the most popular of the modular strategies to handle styling with classes (which is the industry best practice).  It is used by Google, AirBnB, etc, and is my go to way of organizing CSS.